From 69dc5b22e7f31ba091f388577201902aaafea753 Mon Sep 17 00:00:00 2001
From: Petr Oros <poros@redhat.com>
Date: Wed, 17 Sep 2014 16:24:17 -0400
Subject: [redhat] kabi: Adding symbol nla_put (lib/nlattr.c)

Message-id: <5bf43d98e4dd18fb6e0fc0740b4d72ec8ce2f1a7.1410510773.git.poros@redhat.com>
Patchwork-id: 96031
O-Subject: [PATCH RHEL7.1 137/233] kabi: Adding symbol nla_put (lib/nlattr.c)
Bugzilla: 1077335
RH-Acked-by: Jiri Benc <jbenc@redhat.com>

Bugzilla: 1077335
    https://bugzilla.redhat.com/show_bug.cgi?id=1077335

    Description:
    ============
    Netlink API

Signed-off-by: Petr Oros <poros@redhat.com>
---
 create mode 100644 redhat/kabi/kabi-module/kabi_ppc64/nla_put
 create mode 100644 redhat/kabi/kabi-module/kabi_s390x/nla_put
 create mode 100644 redhat/kabi/kabi-module/kabi_x86_64/nla_put

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/redhat/kabi/kabi-module/kabi_ppc64/nla_put  b/redhat/kabi/kabi-module/kabi_ppc64/nla_put 
new file mode 100644
index 0000000..ccaf1f5
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_ppc64/nla_put  
@@ -0,0 +1,2 @@
+#1-
+0x7a2fba58 nla_put vmlinux EXPORT_SYMBOL
diff --git a/redhat/kabi/kabi-module/kabi_s390x/nla_put  b/redhat/kabi/kabi-module/kabi_s390x/nla_put 
new file mode 100644
index 0000000..5dc9e42
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_s390x/nla_put  
@@ -0,0 +1,2 @@
+#1-
+0x9b652f3e nla_put vmlinux EXPORT_SYMBOL
diff --git a/redhat/kabi/kabi-module/kabi_x86_64/nla_put  b/redhat/kabi/kabi-module/kabi_x86_64/nla_put 
new file mode 100644
index 0000000..edf8923
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_x86_64/nla_put  
@@ -0,0 +1,2 @@
+#1-
+0xdc9c9463 nla_put vmlinux EXPORT_SYMBOL
-- 
1.7.1